src/sentry/monitors/models.py

```diff
@@ -74,14 +74,33 @@ class MonitorEnvironmentValidationFailed(Exception):
     pass
 
 
+class MonitorObjectStatus:
+    ACTIVE = 0
+    DISABLED = 1
+    PENDING_DELETION = 2
+    DELETION_IN_PROGRESS = 3
+
+    WAITING = 4
```

Comment:
Let's drop a comment on what this means out-side the context of billing usage? What does it mean for the monitor
```suggestion
    WAITING = 4
    """
    What does this do?
    """
```